### PR TITLE
Feature diversity measures

### PR DESCRIPTION
**Pull request by LaraFuhrmann**
This PR consists of a new feature that is computing a selection of diversity indices based on SNVs. It should be ready to merge.

### CODE REVIEW

File: workflow/rules/common.smk [hunk 1]
```diff
@@ -363,7 +364,7 @@ for p in patient_list:
     consensus.append(os.path.join(sdir, "references/ref_ambig.fasta"))
     consensus.append(os.path.join(sdir, "references/ref_majority.fasta"))
 
-    consensus.append(os.path.join(sdir, "references/consensus.bcftools.fasta"))
+    # consensus.append(os.path.join(sdir, "references/consensus.bcftools.fasta"))
```
Comment (kpj) [inline workflow/rules/common.smk:367]: This should not be commented out, right?

File: workflow/envs/diversity_measures.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - python>=3.6
+  - pyvcf
```
Comment (kpj) [inline workflow/envs/diversity_measures.yaml:7]: Why does `pyvcf` not have a version?

File: workflow/rules/common.smk [hunk 3]
```diff
@@ -417,8 +418,29 @@ for p in patient_list:
...
+    if not config.output["snv"] and config.output["diversity"]:
+        raise RuntimeError(
+            "Cannot generate diversity without calling variants (make sure to set `snv = True`) in config."
+        )
+
+    if config.output["snv"] and config.output["diversity"]:
+        diversity_measures.append(
+            os.path.join(sdir, "variants/SNVs/diversity_measures.csv")
+        )
+
```
Comment (kpj) [inline workflow/rules/common.smk:431]: Is it necessary to include these files if we keep the aggregated one later on?

File: workflow/scripts/compute_diversity_measures.py [hunk 4]
```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env python3
```
Comment (kpj) [inline workflow/scripts/compute_diversity_measures.py:1]: This file should be formatted with `black`.

**Review by kpj — CHANGES_REQUESTED** [review]
Looks very good! I added a few comments.
In addition, could you check that the diversity measures are also computed when the GitHub Actions tests run?

File: workflow/rules/common.smk (hunk 1 above)
Comment (LaraFuhrmann) [reply workflow/rules/common.smk:367]: Yes, you are right.

File: workflow/rules/common.smk (hunk 3 above)
Comment (LaraFuhrmann) [reply workflow/rules/common.smk:431]: No, we don't need to include these files. I removed them.

File: workflow/scripts/compute_diversity_measures.py (hunk 4 above)
Comment (LaraFuhrmann) [reply workflow/scripts/compute_diversity_measures.py:1]: Done. Thanks 👍

File: workflow/envs/diversity_measures.yaml (hunk 2 above)
Comment (LaraFuhrmann) [reply workflow/envs/diversity_measures.yaml]: I added a version now.

**Review by kpj — APPROVED** [review]

**Pull request merged**